app/src/docker/docker-compose.yml

```diff
@@ -157,7 +158,7 @@ services:
       <<: *redis-placeholder-vars
       <<: *postgres-vars
       <<: *mas-vars
-      PERSIST_TRACKING_FOLDER: ${PERSIST_TRACKING_FOLDER}
+      PERSIST_TRACKING_FOLDER: /persist/tracking
```

Comment:
This should not rely on the env var. The folder should be the same as the volume being mounted below.